content/cli-v2.md

```
@@ -872,6 +873,18 @@ See [CPI config](cpi-config.md).
     bosh -e my-env update-cpi-config cpis.yml
     ```
 
+---
+### Deploy config {: #deploy-config-mgmt }
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)